R/build-manual.R

```diff
@@ -0,0 +1,19 @@
+
+#' Create package pdf manual
+#'
+#' @param pkg package description, can be path or package name.  See
+#'   \code{\link{as.package}} for more information.
+#' @param path path in which to produce package manual.
+#'   If \code{NULL}, defaults to the parent directory of the package.
+#'
+#' @seealso \code{\link{RdUtils}}
+#' @export
+
+build_manual <- function(pkg = ".", path = NULL) {
+  pkg <- as.package(pkg)
+  if (is.null(path)) {
+    path <- dirname(pkg$path)
+  }
+  name <- paste0(pkg$package, "_", pkg$version, ".pdf", collapse = " ")
+  system(paste0("R CMD Rd2pdf --force --output=", path, "/", name, " ", shQuote(pkg$path), collapse = " "))
```

Comment:
I made the two other edits you suggested, but I don't understand the syntax for `callr::rcmd` and I can't make it work.